rfcs/2021-28-07-motion-design-core-actions.md

```diff
@@ -0,0 +1,176 @@
+| **Fade Out** | Opacity value moves from a higher to lower.<br>**Ex:** Disappearing of a card from a page.<br>**Unit:** `%` | <img src="images/motion/core-actions/image9.gif" alt="core-action-fade-out" width="200px"> |
+
+### 4. **Fill**
+* **Color tokens:** Predefined Color tokens within the design system and changing or adding different tokens for diffrent state leads to change in the color value as well.
+
+### Types:
+| Type | Description | Example |
+|---|---|---|
+| **Color token change** | When there is a change in state of a UI element, use the fill effect. <br>**Ex:** Hovering over a button.<br>**Unit:** token value (`color.btn.primary.hover`) | <img src="images/motion/core-actions/image10.gif" alt="core-action-fill" width="200px"> |
+
+### 5. **Morph**
+> It can be used on mask
+* **Shape:** A graphics element that is defined by some combination of straight lines and curves. Read more on [Shapes](https://www.w3.org/TR/SVG2/shapes.html). Specifically: circle, ellipse, line, path, polygon, polyline and rect.
+
+### Types:
+| Type | Description | Example |
+|---|---|---|
+| **Shape change** | When a shape gets morhed into another. <br>**Ex:** Opening of dropdown menu.<br>**Unit:** - | <img src="images/motion/core-actions/image11.gif" alt="core-action-morph" width="200px"> |
+
+### **Combination of Properties**
+|  | Scale | Move | Fade | Fill | Morph |
+|---|---|---|---|---|---|
+| **Scale** | ⛔️ | ✅ | ✅ | ❌ | ❌ |
+| **Move** | ✅ | ⛔️ | ✅ | ❌ | ✅ |
+| **Fade** | ✅ | ✅ | ⛔️ | ❌ | ❌ |
+| **Fill** | ❌ | ❌ | ❌ | ⛔️ | ✅ |
+| **Morph** | ❌ | ✅ | ❌ | ✅ | ⛔️ |
+
+### **Examples**
+| Type | **Scale Fade** | **Move Fade** |**Morph Move** | **Scale Move** | **Morph Fill** |
+|---|---|---|---|---|---|
+| Example |<img src="images/motion/core-actions/image12.gif" alt="core-actions-combination-scale-fade" width="150px">|<img src="images/motion/core-actions/image13.gif" alt="core-actions-combination-move-fade" width="150px"> | <img src="images/motion/core-actions/image15.gif" alt="core-actions-combination-morph-move" width="150px"> |<img src="images/motion/core-actions/image14.gif" alt="core-actions-combination-scale-move" width="150px"> | <img src="images/motion/core-actions/image16.gif" alt="core-actions-combination-morph-fill" width="150px"> |
+
+> **Note:** Try not to combine more than 2 properties.
+
+### **Accessibility**
+* For [reduced Motion](https://www.smashingmagazine.com/2020/09/design-reduced-motion-sensitivities/) (for motion sensitivities) we should use Fade action instead of Move.
+
+# Drawbacks/Constraints
+In this, we didn’t consider animation of Icons, Loaders and Typography.
+* Icons and loaders will have a library and designers will not be creating these occasionally, that’s why we didn’t include rotation property in core actions.
```

Comment:
But IMO we shall still add them here since this RFC will be defining our core tokens and everything related to motion

Reply:
Let's explain this in detail @dubeydushyant. As we are only talking about the core actions, we should define the core properties instead of talking about their application.

Reply:
I added the Rotation property, I think I need to update this line also.